pipeline/price_modelling/controller.py

```diff
@@ -1,147 +1,23 @@
-from __future__ import division
 import os
-import pandas as pd
-import numpy as np
 from sqlalchemy import create_engine
-from datetime import timedelta
-from sklearn.linear_model import ElasticNetCV
-from statsmodels.nonparametric.smoothers_lowess import lowess
+import pandas as pd
+
+forecast_period = 90
```

Comment:
Probably you already told me, but have you already tested different lags?
Should we do, after the meeting, a proper test to tune this?

Reply:
I have, but it's inconclusive. I guess the question is how far into the future with what level of uncertainty is a good mix for the purpose of the prediction.

But this is something that we do need to consider and adjust later.